sor10874/readmes/25-30 (#725)
Esri/arcgis-runtime-samples-android: java/identify-kml-features/README.md

1. Create an `OnTouchListener` on the `MapView`.
2. On tap:
  * Dismiss the `Callout`, if one is showing.

>   * Call `identifyLayersAsync(...)` passing in the `KmlLayer`, screen point and tolerance.

Reviewer: Layer should be singular in the method call, and it probably makes sense to include the class name as well.
```suggestion
  * Call `MapView.identifyLayerAsync(...)` passing in the `KmlLayer`, screen point and tolerance.
```

  * Await the result of the identify and then get the `KmlPlacemark` from the result.
  * Create a callout at the calculated map point and populate the callout content with text from the placemark's `BalloonContent`. NOTE: KML supports defining HTML for balloon content and may need to be converted from HTML to text.
  * Show the callout.
  
Note: There are several types of KML features. This sample only identifies features of type `KmlPlacemark`.

## Relevant API

> * GeoView.identifyLayerAsync

Reviewer: API shouldn't include method names
```suggestion
* GeoView
```

* IdentifyLayerResult
* KmlLayer
* KmlPlacemark

> * KmlPlacemark.getBalloonContent

Reviewer: ```suggestion
```
